calcom/cal.com
# fix: More instances of wrong avatar URL fixed

## What does this PR do?

Fixed avatar in users listing for Admin view of users
![image](https://github.com/calcom/cal.com/assets/1780212/2a7dadbf-f764-48ac-800d-3e26fb26f799)

Fixed avatar, when a team installable app is installed for the user
![image](https://github.com/calcom/cal.com/assets/1780212/78a1894a-7701-4ff6-a060-8d00d26e6c79)


## Type of change

## How should this be tested?
- Go to http://app.cal.local:3000/settings/admin/users and check avatar for an org member
- Login as an org member and install jitsi for one of your teams as well as yourself and then go to http://app.cal.local:3000/apps/installed/conferencing and check your avatar there.

## Mandatory Tasks


## Checklist
- I haven't added tests that prove my fix is effective or that my feature works
